samza-core/src/main/java/org/apache/samza/operators/impl/WindowOperatorImpl.java

```diff
@@ -21,14 +21,23 @@
 package org.apache.samza.operators.impl;
 
 import com.google.common.base.Preconditions;
-import org.apache.samza.config.Config;
+import java.util.ArrayList;
```

Comment:
Prefer either reverting import order changes, or if this is the order in majority of classes, doing this for all files in the module for consistency (i.e., not just the ones touched in this PR). If the latter, would also prefer to pick the default IntelliJ sort order and publish it as part of the contributor's guidelines / code style guidelines. We can do the rest of the classes at once in a follow up PR.

Reply:
I reverted my import order changes to the default. There are still some classes which already don't match the default, so I generally tried to leave those alone for now.